### DISCUSSION TITLE
v3.5.1

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#9331] [General]
Maintenance v3.5 release

https://github.com/gitextensions/gitextensions/blob/release/3.5/GitUI/Resources/ChangeLog.md#version-351-04-jul-2021

<hr /><em>This discussion was created from the release <a href='https://github.com/gitextensions/gitextensions/releases/tag/v3.5.1'>v3.5.1</a>.</em>

**Comment by bmills74** [comment]
Call me an old dog, but reversing the colour styling, now green for local, red for remote, has hurt my brain!

**Reply by gerhardol** [reply]
The change was done to conform with git-log colors
(There quite some discussion about this...)
Use your own theme to revert the changes.

**Reply by jrfnl** [reply]
Also see #9113

**Comment by VsevolodGolovanov** [comment]
After updating to extensions `3.5.1.12196` and git `2.32.0.windows.2` I'm getting these errors in the Diff window:
![image](https://user-images.githubusercontent.com/8070317/125972582-586b06aa-fbfd-4880-abcf-b7b67f9e59df.png)

Is this strictly a git error, not extensions? Is there a fix? Should I rollback to an older version? Which one?

**Comment by pmiossec** [comment]
Yes it seems to be a git error. `size_t` is common in C language used by
git.

Try the previous version of Git.

**Comment by VsevolodGolovanov** [comment]
Same deal with 2.31.1-32.
But 2.30.2-32 works.

**Reply by pmiossec** [reply]
OK. So if you want a good reason, you will have to ask in the git for windows repository. 

Is there a specific reason to use a 32bits version now that all the OS are 64bits since a long time?

**Reply by VsevolodGolovanov** [reply]
No reason other than inertia.
